utils/wui.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+
+import sys
+
+import os
+from pathlib import Path
+import gzip
+
+
+def generate_header():
+    print('#include <lwip/apps/fs.h>')
+    print('#include <lwip/def.h>')
+
+    print('#define file_NULL (struct fsdata_file *)NULL\n')
+
+    print('#ifndef FS_FILE_FLAGS_HEADER_INCLUDED')
+    print('#define FS_FILE_FLAGS_HEADER_INCLUDED 1')
+    print('#endif')
+    print('#ifndef FS_FILE_FLAGS_HEADER_PERSISTENT')
+    print('#define FS_FILE_FLAGS_HEADER_PERSISTENT 0')
+    print('#endif')
+    print('#ifndef FSDATA_FILE_ALIGNMENT')
+    print('#define FSDATA_FILE_ALIGNMENT 0')
+    print('#endif')
+    print('#ifndef FSDATA_ALIGN_PRE')
+    print('#define FSDATA_ALIGN_PRE')
+    print('#endif')
+    print('#ifndef FSDATA_ALIGN_POST')
+    print('#define FSDATA_ALIGN_POST')
+    print('#endif')
+    print('#if FSDATA_FILE_ALIGNMENT == 2')
+    print('#include "fsdata_alignment.h"')
+    print('#endif\n')
+
+
+def to_hex(data):
+    return ','.join(map(lambda b: hex(b), data)) + ','
```

Comment:
No need for this. See [bytes.hex()](https://docs.python.org/3/library/stdtypes.html#bytes.hex)

Reply:
I'm afraid not. This thing also adds the separators (the commas) between the values. The `sep` support got added in python 3.8 and we claim in README.md that we support python 3.6.